ShittyLinqTests/ShittyTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace ShittyLinqTests
+{
+  [TestClass]
+  public class ShittyLinqTests
+  {
+    IEnumerable<int> range;
+
+    [TestInitialize]
+    public void Setup()
+    {
+      range = Enumerable.Range(1, 10);
+    }
+
+    [TestMethod]
+    public void TestAggragate()
```

Comment:
Small nitpick: The spelling of the test method name should match the spelling of the method being tested.
TestAggragate -> TestAggregate

Reply:
Heh, agreed